### ISSUE TITLE
Jellyfin Media Player does not follow web release as indicated by 1.11.0 release notes

### ISSUE THREAD

**Issue by TheRogueChivalry** [jellyfin/jellyfin-media-player#716]
**Describe the bug**
The Jellyfin Media Player does not follow the web release. 1.11.0 release notes indicates it should. Updated server to 10.9.7 and the web is still 10.9.6 when I go to the dashboard through the player.

**Expected behavior**
The feature indicated above should work as it states in the release notes.

**Screenshots**
![image](https://github.com/jellyfin/jellyfin-media-player/assets/128194421/0cb90ca2-1a17-4e54-9eee-f0bf78f933b5)

**Desktop (please complete the following information):**
 - OS: Windows 11 Education 64bit
 - Version 23H2
 - Installation Method Windows 64bit Installer 
 
 Jellyfin Client: 1.11.1

**Comment by TheRogueChivalry** [comment]
> How long has it been since you updated to 10.9.7? in my experience, the update in Media Player doesn't happen instantly, it takes a while, perhaps because it saves things in a cache? and after a while the cache expires, or renews itself from time to time. 

Well to see if there was an issue along the lines you were saying:

1. I removed the Jellyfin Media Player totally including temp data. 

2. I reinstalled the client, logged in again and its the same thing.

Lingering data is not the issue. 

The feature I am talking about is very recent and is intended to allow for delays in updating of the media player in favor of using the external web client. For example, there was a graphical glitch in chapter markers where they would "rerender" or "blink". The media player client was out of date and the issue was not resolved until the next time the client was updated. Meanwhile jellyfin-web had this already patched and you would not have an issue if you used a browser or any clients that used the external web client. 

The client as of 1.11.0 should not have this issue. It should be using the external web client.

**Comment by Uncle-Tio** [comment]
I did the update and tested it, I had the same result as you, but I deleted the folder `C:\Users\USERNAME\AppData\Local\Jellyfin Media Player` (note that there are 2, it's the one with spaces), and it worked. Of course, it's not a solution, just a bandage, but I've been using the external web since it became an option, and eventually the "update" happened on the client without having to delete anything.

If it's still not updating for you, does the browser return as web 10.9.7?
